src/Swagger/Serializer/DocumentationNormalizer.php

```diff
@@ -518,8 +518,9 @@ private function getPropertySchema(PropertyMetadata $propertyMetadata, \ArrayObj
             $propertySchema['description'] = $description;
         }
 
+        $swaggerContext = $propertyMetadata->getAttributes()['swagger_context'] ?? [];
         if (null === $type = $propertyMetadata->getType()) {
-            return $propertySchema;
+            return new \ArrayObject((array) $propertySchema + $swaggerContext);
```

Comment:
Can't we directly initialize `propertySchema` as an array line 511 and create directly the final `\ArrayObject` here?

Reply:
@dunglas I think I fixed it, not sure this is exactly what you meant so please check.